src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,125 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final String DEFAULT_LOGIN = "login";
+    private static final String DEFAULT_PASSWORD = "123456";
+    private static final int DEFAULT_AGE = 18;
+    private static final int NOT_VALID_AGE = 17;
+    private static final int NEGATIVE_NUMBER = -1;
+    private static RegistrationService registrationService;
+    private static User firstUser;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        firstUser = new User();
+        firstUser.setLogin(DEFAULT_LOGIN);
+        firstUser.setPassword(DEFAULT_PASSWORD);
+        firstUser.setAge(DEFAULT_AGE);
+    }
+
+    @Test
+    void register_nullUser_NotOk() {
```

Comment:
```suggestion
    void register_nullUser_notOk() {
```
The first letter of each part of the method name must be in lower case